playbooks/installation/disconnected_satellite6.adoc

```diff
@@ -0,0 +1,141 @@
+7.  TODO - add activation key to content view.
+
+== Create an Activation Key and add the Content View to it
+
+Creating an activation key isn't strictly necessary, but it will come in handy since it will reduce the number of steps required for each host you register to the Satellite (each host that requires the access to the Registry, at least).  Be sure to use an alternative key for those hosts you don't want to be able to access the Registry.
+
+1.  Click the "Content" pull-down and then "Activation Keys".
+2.  Click the "New Activation Key" button.
+3.  Give your key a useful name.  Keep in mind it will be used later.  For example, "dockerActKey".
+4.  Select the appropriate Environment(s) and then finally the Content View when it populates.
+5.  Click "Save".
+
+
+== Register OpenShift 3 Nodes to Disconnected Satellite
+
+Now we'll proceed to subscribe our clients to the newly-created and populated Satellite Server.
+you will need to download the katello consumer packages from the server, and install the rpms.
+
+ curl -O --insecure https://your.satellite.instance/pub/katello-ca-consumer-latest.noarch.rpm
+ yum localinstall katello-ca-consumer-latest.noarch.rpm
+
+
+Download and save the katello server public cert to the default location of /usr/share/rhn
+
+ curl -O --insecure https://your.satellite.instance/pub/katello-server-ca.crt
+
+
+From here you can run subscription-manager as usual to download the ose3 packages and theoretically follow the same installation procedure as a connected server.  Depending on how your katello-ca-consumer-latest.noarch RPM was built, you may need to edit /etc/rhsm/rhsm.conf on your Nodes to point to the correct server.  Typically this should be handled by the RPM's install.
+
+ subscription-manager register --username=jdoe --password=yourpass --org="IT Dept" --activationkey=dockerActKey
+ The system has been registered with ID: 37f97705-762f-4594-bf6a-d0cdc73a3f79
+
+This command uses the Activation Key we setup earlier - so you'll need minimal configuration from here on out.
+
+Depending on your specific organization setup in Satellite, you may need additional steps to get repositories showing up.  Use the [Satellite 6.1 Docs] (https://access.redhat.com/documentation/en-US/Red_Hat_Satellite/6.1/) to follow the necessary steps to do so.
+
+Once packages are available, install docker and setup docker storage.
+
+ yum install -y docker
+
+
+== Issues / Workarounds
+* Satellite 6.1 was unable to pull the OSE3 images from the docker repo using a plain `pull` command. This is due to a problem with Satellite interpreting docker labels. You must use a `pull -a` docker command (pull all) to retrieve the OSE3 images.
+* The Docker tab on Satellite 6.1 never had any images listed within it no matter which manifest we put in it. Ticket https://engineering.redhat.com/rt/Ticket/Display.html?id=388088[#388088] was opened to track it.
```

Comment:
Since this is a public site, if there is any way we can point to a BZ here instead, that would be best.